src/work-allocation/containers/task-assignment-confirm/task-assignment-confirm.component.spec.ts

```diff
@@ -7,7 +7,7 @@ import { RouterTestingModule } from '@angular/router/testing';
 import { SessionStorageService } from '@hmcts/ccd-case-ui-toolkit/dist/shared/services';
 import { ExuiCommonLibModule } from '@hmcts/rpx-xui-common-lib';
 import { Observable, of, throwError } from 'rxjs';
-import { PersonRole } from '../../../../api/workAllocation2/interfaces/person';
+import { PersonRole } from '../../../../api/workAllocation/interfaces/person';
```

Comment:
Should not use api - should replicate on src

Reply:
Scope of "Removing MVP" is not finetuning previously developed code , explained in detail on another comment.